tests/recipes/click-test-feedstock/recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   number: {{ build_num }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: python -m pip install .
```

Comment:
This change is not needed and is unrelated to the packaging of conda-smithy. Please back it out.

Reply:
yup, hence me raising these cases in the PR body, thanks for clarifying! I made these separate commits so I can revert them easily.

Reply:
I’ve done reverts for now, but I was wondering if you want all these as separate commits, or should I rebase these
- [ ] into a single comit
- [ ] separate commits are good, just take out the original commits instead of a revert

Reply:
commit hygene doesn't bother me - we can squash on merge

Reply:
Thanks!